test/test_git_deploy.go

```diff
@@ -265,11 +265,20 @@ func (s *GitDeploySuite) TestSlugbuilderLimit(t *c.C) {
 	r := s.newGitRepo(t, "slugbuilder-limit")
 	t.Assert(r.flynn("create"), Succeeds)
 	t.Assert(r.flynn("env", "set", "BUILDPACK_URL=git@github.com:kr/heroku-buildpack-inline.git"), Succeeds)
-	t.Assert(r.flynn("limit", "set", "slugbuilder", "memory=500MB"), Succeeds)
+	t.Assert(r.flynn("-a", "gitreceive", "env", "set", "SLUGBUILDER_DEFAULT_MEMORY_LIMIT=2GB"), Succeeds)
```

Comment:
Hmm, I guess if a deploy was taking longer than the deploy timeout for gitreceive. I'll switch it to a non-parallel suite.